reflex/utils/prerequisites.py

```diff
@@ -367,25 +367,32 @@ def install_bun():
     )
 
 
-def install_frontend_packages():
-    """Installs the base and custom frontend packages."""
+def install_frontend_packages(packages: List[str]):
+    """Installs the base and custom frontend packages.
+
+    Args:
+        packages (List[str]): A list of package names to be installed.
+
+    Example:
+        >>> install_frontend_packages(["react", "react-dom"])
+    """
     # Install the base packages.
     process = processes.new_process(
         [get_install_package_manager(), "install", "--loglevel", "silly"],
         cwd=constants.WEB_DIR,
         shell=constants.IS_WINDOWS,
     )
+
     processes.show_status("Installing base frontend packages", process)
 
-    # Install the app packages.
-    packages = get_config().frontend_packages
+    # Install the custom packages, if any.
     if len(packages) > 0:
         process = processes.new_process(
             [get_install_package_manager(), "add", *packages],
             cwd=constants.WEB_DIR,
             shell=constants.IS_WINDOWS,
         )
-        processes.show_status("Installing custom frontend packages", process)
+        # processes.show_status("Installing custom frontend packages", process)
```

Comment:
```suggestion
        processes.show_status("Installing custom frontend packages", process)
```

This line is causing a race condition! Because the `bun add ...` command doesn't have `run=True` set, it starts the command and returns the running process object immediately. Usually bun is pretty fast, but if it's still installing packages after this function returns, then occasionally node will hit a compilation error due to missing imports.

`processes.show_status` is a blocking call that waits for the command to exit, so we previously were waiting here, but with this line commented out, the code is essentially starting the process and not waiting for it to finish.